lib/model/model.go

```diff
@@ -100,7 +102,8 @@ type Model struct {
 	pmut                sync.RWMutex                   // protects the above
 }
 
-type folderFactory func(*Model, config.FolderConfiguration, versioner.Versioner, *fs.MtimeFS) service
+type folderFactory func(*Model, config.FolderConfiguration, versioner.Versioner,
+	*fs.MtimeFS, <-chan []string) service
```

Comment:
Single line. Also, I think we should pass the fswatcher.Service oppose to some random channel that doesn't explain what it is.